android/app/build.gradle

```
@@ -231,6 +238,38 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
+    flavorDimensions "version"
+
+    productFlavors {
+        prod {
+            minSdkVersion rootProject.ext.minSdkVersion
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            resValue "string", "app_name", "ReactNativeBase"
+        }
+        develop {
+            applicationIdSuffix '.dev'
+            minSdkVersion rootProject.ext.minSdkVersion
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            versionNameSuffix "- Develop"
+            resValue "string", "app_name", "RNBase - Develop"
+        }
+        qa {
+            applicationIdSuffix '.qa'
+            minSdkVersion rootProject.ext.minSdkVersion
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            versionNameSuffix "- QA"
+            resValue "string", "app_name", "RNBase - QA"
+
```

We did this on the Special Olympics project, we can set the `versionCode` and `versionName` for each flavor and remove them from the `defaultConfig`.

```suggestion
        versionCode 1
        versionName "1.0"
```